docs/metadata.md

```diff
@@ -0,0 +1,14 @@
+# Volumetric metadata about the VDS
+
+Retrieve basic information related to the VDS file such as axis definitions, overall shape and size, coordinate system, import time stamp and source filename. 
```

Comment:
Maybe not VDS file, but VDS data?

Some of the information we return doesn't belong to vds file - we calculate it.